clustintime/clustintime.py

```diff
@@ -110,7 +111,7 @@ def clustintime(
 
     print("Mask applied!")
     # Create data
-    if timings == True:
+    if timings:
```

Comment:
If `timings` is truthy, no need to compare it to `True`. Flake complained about this.